lib/kino_bumblebee/task_cell.ex

```diff
@@ -907,7 +907,22 @@ defmodule KinoBumblebee.TaskCell do
 
   @impl true
   def to_source(attrs) do
-    for quoted <- to_quoted(attrs), do: Kino.SmartCell.quoted_to_string(quoted)
+    for quoted <- to_quoted(attrs) do
+      quoted
+      |> remove_empty_blocks()
+      |> Kino.SmartCell.quoted_to_string()
+    end
+  end
+
+  defp remove_empty_blocks(ast) do
```

Comment:
@josevalim let me know if there's a better way to unquote something that doesn't end up in the formatted string :)

Reply:
I think you could use `unquote_splicing(…)`. If it is an empty list, it is no-op. Otherwise a list of expressions. WDYT?

If you want to keep it as this, I would change it to match? instead of == so you can ignore the metadata.

Reply:
YESS fantastic, thanks!